docs/UserGuide.md

```diff
@@ -425,6 +425,15 @@ Clears all entries from TaskHub.
 
 Format: `clear`
 
+<div markdown="block" class="alert alert-danger">
+
+**:no_entry: Danger**<br>
+
+* This action is irreversible.
+* Please ensure that you are certain that you want to all entries before proceeding.
```

Comment:
The word `delete` to be added. Maybe it would be better to specify that all employees, projects and tasks will be deleted?

Reply:
Done